features/dashboard.feature

```diff
@@ -0,0 +1,183 @@
+Feature:
+  As a referent, animator or simple adherent
+  In order to see all my informations
+  I should be able to acces my dashboard
+
+  Background:
+    Given the following fixtures are loaded:
+      | LoadAdherentData        |
+      | LoadTurnkeyProjectData  |
+      | LoadCitizenProjectData  |
+      | LoadEventData           |
+
+  # Adherent
+  Scenario: As an adherent, I can see information about me
```

Comment:
IMO we can group scenarios by adherents / referents / animator / etc.. no ? 
We can gain for example the step call `Given I am logged as..`  and the `I am on...`

Reply:
Yes and put the route in background also

Reply:
saw it together, done